### DISCUSSION TITLE
Add more meaningful prompts instead of default `>`

### DISCUSSION THREAD

**Discussion by Delapouite** [PatrickF1/fzf.fish#275] [Archive] [answered]
Hello

Currently, all defined commands use the default `>` prompt. Sometimes this minimalism can be confusing.

For example if your shell is currently in a directory with a git repo with an empty status, pressing `Ctrl+Alt+S` triggers the `_fzf_git_search_status` function as expected. It displays a fzf window with just a `0/0` counter, a blank candidates list and preview and no real indication about what is supposed to happen. For someone activating this by accident, it provides no clue about what's going on.

Therefore, I suggest that the commands provided by this great plugin could have dedicated prompts. In the example described above it could be `git status >`, so the user is not lost anymore.

Thanks.

**Comment by PatrickF1** [comment]
Hi Bruno, that's a great idea! Instead of git status, I'll probably put Search Git Status, in line with the canonical names (#270). What do you think? Since I rewrote the documentation, do you think users will immediately understand the canonical names?

**Comment by Delapouite** [comment]
Yes I agree, canonical names would be even more descriptive.

**Comment by PatrickF1** [comment, marked as answer]
Thanks for your great suggestion. This is done now: https://github.com/PatrickF1/fzf.fish/commit/3666395bc10752c7afd21210fe5c3458d0f502bf

**Comment by seowalex** [comment]
Is there a way to configure this behaviour?

**Reply by PatrickF1** [reply]
Sure! Pass in `--prompt` using https://github.com/PatrickF1/fzf.fish#pass-fzf-options-for-a-specific-command
